src/Main/Main.js

```diff
@@ -1,5 +1,10 @@
 import * as CalendarUtils from '../utils/CalendarUtils/CalendarUtils';
+import State from '../resources/State/State';
 
 export const processParams = async (container, proxyServerUrl) => {
-  await CalendarUtils.initializeBasicAppearance(container, proxyServerUrl);
+  const state = new State(container, proxyServerUrl);
+
+  state.render();
```

Comment:
If we'll call this state it shouldn't know how to render. So it's either something else, or (and i like the separation here) we have something that can render a state. (which is already the case actually, it's just used from the state)

Reply:
Regarding this proposal, could you check my [**comment**](https://github.com/c-hive/team-contribution-calendar/pull/13#discussion_r276108299) below?